src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/AccessToken/TokenHandlerFactoryInterface.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\SecurityBundle\DependencyInjection\Security\AccessToken;
+
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * TokenHandlerFactoryInterface is the interface for all token handler factories.
+ *
+ * @author Vincent Chalamon <vincentchalamon@gmail.com>
+ */
+interface TokenHandlerFactoryInterface
+{
+    /**
+     * Create a generic token handler service.
```

Comment:
```suggestion
     * Creates a generic token handler service.
```

s-form everywhere please. Not sure about `generic` also